auto-instrument-handler/index.js

```diff
@@ -21,19 +21,22 @@ const getHandlerAsync = async () => {
   );
 };
 
+export const removeLumigoFromError = (stacktrace) => {
+  const stackArr = stacktrace.split('\n');
+
+  const patterns = ['/dist/lumigo.js:', 'auto-instrument'];
+  const cleanedStack = stackArr.filter(v => !patterns.some(p => v.includes(p)));
+
+  return cleanedStack.join('\n');
+}
+
```

Comment:
Updated for now just to get the ball rolling @harelmo-lumigo

Reply:
Cool, I see now why it has to be replicated :( (though we can change this in the future, but definitely not as a part of this PR)